README.md

```diff
@@ -1 +1,70 @@
-# terraform-google-cloud-dns
\ No newline at end of file
+# terraform-google-cloud-dns
+
```

Comment:
Let's add a `## Description` field to provide the use case for this module, and possibly discuss the reasonings around using individual resources rather than using lists and running into Terraform's undesirable behavior around recreating resources when a list of resources is provided.